### PR TITLE
CTA for testingjavascript.com

### PR DESCRIPTION
**Pull request by vojtaholik**
This adds a testingjavascript.com CTA on the home page and at the end of every blog post that contains "testing" keyword. It uses `react-responsive-carousel (27.6 kB)` to loop through all artworks.

### CODE REVIEW

**Review by kentcdodds — APPROVED** [review]
Awesome!

File: src/components/header.js [hunk 1]
```diff
@@ -17,7 +18,11 @@ function HeaderLink({headerColor, ...props}) {
         textDecoration: 'none',
         color: headerColor ? headerColor : theme.colors.body_color,
         '&:hover,&:focus': {
-          background: 'rgba(40, 28, 77, 0.2)',
+          background:
+            headerColor === theme.colors.white
+              ? 'rgba(40, 28, 77, 0.3)'
+              : lighten(0.4, theme.brand.primary),
+          //background: lighten(0.4, theme.brand.primary),
```
Comment (apoorvtomar2222) [inline src/components/header.js:25]: @vojtaholik  Is it any benefit to leave commented code here? just a query

cc @kentcdodds

Comment (vojtaholik) [reply src/components/header.js:25]: whoops, thanks for pointing that out Apoorv, fixed in https://github.com/kentcdodds/kentcdodds.com/commit/fd1a6d4b381caf5c4021e45eb3b58ecde2f2784a

Comment (apoorvtomar2222) [reply src/components/header.js:25]: Thanks for a quick response.

**Pull request merged**